README.md

```diff
@@ -82,8 +82,6 @@ If at any point, your database becomes corrupted or you no longer have `db.sqlit
 ## Dependencies & Libraries
  * **[Django Rest Framework](http://www.django-rest-framework.org/) v. 3.8.2** - A Django library that provides powerful authentication protocols.
  * **[Django Simple History](https://django-simple-history.readthedocs.io/en/latest/) v. 2.0** - A Django library that provides methods to store changes to models, and thus, is used to create app-specific event, org. feeds.
-  * [react-avatar-editor](https://www.npmjs.com/package/react-avatar-editor) - Similar to Facebook profile picture. Allows you to crop, resize, and rotate an uploaded image. Use for event image upload in event creation.
```

Comment:
If we are not going to be using react-avatar-editor anywhere in the code anymore perhaps we should update package.json as well?

Reply:
it's okay I can handle it in a separate PR